package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "vscode-commons",
   "displayName": "Red Hat common extension",
-  "description": "",
+  "description": "Telemetry Middleware",
   "version": "0.0.1",
   "publisher": "redhat",
-  "segmentWriteKey": "MXM7iv13sVaCGqOhnQEGLZxhfy6hecYh",
+  "segmentWriteKey": "Ga2frlNQ8nsYjF8PEGsbdjhCy3VJdncB",
   "engines": {
-    "vscode": "^1.37.0"
+    "vscode": "^1.51.0"
   },
   "categories": [
     "Other"
   ],
   "activationEvents": [
-    "onCommand:vscodeCommons.clearStateAndSettings"
+    "onCommand:vscodeCommons.showTelemetryStatus"
   ],
-  "main": "./out/extension.js",
+  "main": "./dist/extension.js",
   "contributes": {
     "commands": [
       {
-        "command": "vscodeCommons.clearStateAndSettings",
-        "title": "Clear context.globalState and VS Code settings"
```

Comment:
we might want to keep that test command for now, for testing purposes, remove it later once we're getting closer to  releasing

Reply:
Added it back